src/pricing.njk

```diff
@@ -4,42 +4,157 @@ align: center
+        </p>
     </div>
-    <div class="pricing-tile pt-4 pb-6 px-8">
-        <div class="flex flex-col justify-between items-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block">
-            <h4>Premium</h4>
-            <div>
-                <h2>$15</h2>
-                <label class="sub-label">per Node-RED instance/month</label>
+    <div class="pricing-tiles flex flex-col mb-16 md:grid md:flex-row md:max-w-screen-lg">
+        <div class="pricing-tile pt-8 pb-6 md:h-full">
+            <div class="flex flex-col justify-between items-center text-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block md:text-left md:i md:items-left">
+                <h4 class="mt-2 pt-4" style="display:inline-block;">Starter </h4> <h5 style="display:inline-block;">(Open Source)</h5>
+                <label class="label">Self-managed</label>
+                <div class="mt-6 items-center md:text-left md:items-left">
+                    <h2>Free</h2>
+                </div>
+                <p class="mt-6 pb-2"><span class="inline font-normal">Open-Source</span> version of the platform, with deployment options including Docker and Kubernetes.</p>
+                </div>
+                <div class="flex flex-col items-center">
+                <a class="ff-btn ff-btn--secondary-outlined uppercase align-baseline w-full" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>  
+            </div>
+            <div class="flex flex-col pb-6 mt-2 border-b border-gray-700 text-teal-300 text-sm">
+                <note class="note" style="display: inline-block;">Set up time &lt; 30 min</note>
+            </div>
+            <div class="flex flex-col content-start features text-sm font-bold mt-6">
+                <note class="font-bold text-teal-100" style="display: inline-block;">Top features:</note>
+                <ul class="ff-checklist font-light text-white leading-7 mt-2">
+                    <li>Hosting Node-RED</li>
+                    <li>TLS/SSL</li>
+                    <li>Subdomain</li>
+                    <li>Backups (snapshots)</li>
+                </ul>
             </div>
         </div>
-        <div class="flex flex-col sm:flex-row items-center gap-4 mt-4">
-            <div class="sm:w-6/12">
-                <h5>FlowForge Cloud</h5>
-                <p class="mt-2">Get started in less than a minute, fully managed, fully secure.</p>
-                <a class="ff-btn ff-btn--primary uppercase" href="https://app.flowforge.com/account/create" onclick="capture('cta-join', {'position': 'primary'})">Free Trial</a>
+
+            <div class="pricing-tile pb-6 md:h-full middle">
+                <div class="h-8 flex items-center justify-center text-center text-xs font-semibold text-white bg-gray-600 w-auto -mx-7 rounded-t-lg">  
+                    Production ready
```

Comment:
Looking at this, it feels a bit like the other tiers aren't production ready, what do you think?

Reply:
"Get started instantly" works!